jbt-it/neomdb
# 165 client projectoverview

Added an the ProjectOverview page. The page is structured into tabs. One tab for tendered projects, one for the projects the user is staffed on, one to display the table of all projects and one for billing if the user has the correct permission.

How to check:
- navigate to http://localhost:3000/projekte
- check the tabs
- activate the filtes on the "Alle Projekte" tab
- check if the search with a search term is working
- check if the date filtes are working
- check if the core competencies filter is working
- check if the resetting of the filters is working
- check if the deselecting and selecting of all core competencies is working
- check if the sorting of each column is working correctly
